nautobot/ui/src/components/paginator_form.js

```diff
@@ -7,6 +7,9 @@ export default function PaginatorForm({ start, end, total_count }) {
     let paginator_string = `Showing ${start} - ${end} of ${total_count}`;
     function onPageSizeChange(event) {
         let offset = searchParams.get("offset");
+        document
+            .getElementById("ObjectListContainer")
```

Comment:
I wonder if we can access, or have one passed in, a parent to scroll to instead of having to use a hardcoded ID.

Reply:
I can do
```
document
            .getElementsByClassName("chakra-table")
            .scrollIntoView({ alignToTop: true });
```
Which is functionally the same when we have **one and only one** `ObjectListTable` present on the page if that is better.

Reply:
If you set this: https://github.com/nautobot/nautobot/blob/b811c5a346ad3dcf60acba5341e0ae5c3a6cbe7c/nautobot/ui/src/components/ObjectListTable.js#L191-L194 to something like `const TheTable` you could pass that into Paginator (`<Paginator table={TheTable} #...`) and then you should be able to fire `table.scrollIntoView()`.

Reply:
It seems like passing a react component as a prop does not work here. Because `scrollIntoView`() is not a function of the react component. Instead, I used the `useRef` react hook, which does not need a hard-coded ID and is also the standard way to do this in react.
https://github.com/nautobot/nautobot/pull/3754/commits/b98fe1269b618be54edbaa7266a3c7bdb276d997